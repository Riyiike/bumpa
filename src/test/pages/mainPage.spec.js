import React from "react";
import { shallow } from 'enzyme'
import MainPage from "../../pages/mainPage";

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useOutletContext: () => [],
    useSearchParams: () => [],

}))

jest.mock('react', () => ({
    ...jest.requireActual('react'),
    useContext: () => ({
        regionIndex: 1,
        searchFilter: ''
    }),

}))


describe('test mainPage', () => {
    it('test mainPage', () => {
        const btn = shallow(<MainPage />)
        expect(btn).toMatchSnapshot();
    })
})