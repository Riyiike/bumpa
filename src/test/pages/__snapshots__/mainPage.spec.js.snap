// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test mainPage test mainPage 1`] = `
<div
  className="pt-8 px-5 max-w-7xl mx-auto"
>
  <div
    className="sm:flex justify-between mb-5 items-center"
  >
    <SearchInput />
    <DropDown />
  </div>
  <Suspense
    fallback={<MainSuspenceLoading />}
  />
</div>
`;
